backoffice/src/components/layout/SettingsDrawer.tsx

```diff
@@ -141,22 +142,6 @@ export const SettingsDrawer: React.FunctionComponent<ISettingsDrawerProps> = ({
               downloadStarted={handleDownloadStarted}
               downloadComplete={handleDownloadComplete}
             />
-            {/* TODO - Fix loading issue, remove button for now. */}
```

Comment:
I know this is currently commented out, but probably worth leaving in for now as they'll want it again soon